Mathlib/Data/Nat/PartENat.lean

```diff
@@ -652,9 +649,9 @@ open scoped Classical
 theorem toWithTop_add {x y : PartENat} : toWithTop (x + y) = toWithTop x + toWithTop y := by
   refine PartENat.casesOn y ?_ ?_ <;> refine PartENat.casesOn x ?_ ?_
   -- Porting note: was `simp [← Nat.cast_add, ← ENat.coe_add]`
-  · simp only [add_top, toWithTop_top', _root_.add_top]
-  · simp only [add_top, toWithTop_top', toWithTop_natCast', _root_.add_top, forall_const]
-  · simp only [top_add, toWithTop_top', toWithTop_natCast', _root_.top_add, forall_const]
+  · simp only [add_top, toWithTop_top', IsTopAbsorbing.add_top]
+  · simp only [add_top, toWithTop_top', toWithTop_natCast', IsTopAbsorbing.add_top, forall_const]
+  · simp only [top_add, toWithTop_top', toWithTop_natCast', IsTopAbsorbing.top_add, forall_const]
```

Comment:
This doesn't look like an improvement

Reply:
Indeed, it's not needed, I just removed it